languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -66,7 +67,8 @@ object ImportResolver {
 
   case class DirectoryResolver(directory: Path,
                                dontEscapeFrom: Option[String] = None,
-                               customName: Option[String]) extends ImportResolver {
+                               customName: Option[String],
+                               deleteOnClose: Boolean = false) extends ImportResolver {
```

Comment:
Just OOI, where is this not true (it seems like this was a deliberate choice)?

Reply:
Because my impression is that most directory resolvers are instantiated around a directory of WDLs a user created themselves, and we don't want to delete those

Reply:
Right.... very good point!